### PR TITLE
2018 migration fixes

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
not sure how ci didn't detect any of those

### CODE REVIEW

File: gdk4/src/lib.rs [hunk 1]
```diff
@@ -12,12 +12,11 @@
...
+extern crate bitflags;
+extern crate cairo;
 extern crate gdk_pixbuf;
 extern crate gio;
-#[macro_use]
 extern crate glib;
-extern crate bitflags;
-extern crate cairo;
 extern crate libc;
 extern crate pango;
```
Comment (sdroege) [inline gdk4/src/lib.rs:21]: Hrm you shouldn't have any `extern crate` at all afterwards :) They're redundant with Rust 2018

Comment (bilelmoussaoui) [reply gdk4/src/lib.rs]: Oh I wasn't aware of that, thanks!

**Comment by bilelmoussaoui** [comment]
This turned out to be more of a generic cleanup/fixes :)

File: gdk4/src/lib.rs (hunk 1 above)
Comment (sdroege) [reply gdk4/src/lib.rs]: Just like `use $crate;` it's not needed anymore. Dependencies are always in scope

**Pull request merged**